### PR TITLE
Add a .env_sample file, update gitignore, update README.md

### PR DESCRIPTION
**Pull request by thepriefy**
issue #597

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-csharp?pullRequest=599) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-csharp?pullRequest=599) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-csharp?pullRequest=599) it.</sub>

File: README.md [hunk 1]
```diff
@@ -103,6 +103,24 @@ For sample implementations, see the [.NET Core Example](https://github.com/sendg
 
 - Please see the [.nuspec file](https://github.com/sendgrid/sendgrid-csharp/tree/master/nuspec).
 
+## Setup Environment Variables
+
+### Initial Setup
+
+```bash
+cp .env_sample .env
```
Comment (mbernier) [inline README.md]: I think this should be `COPY` as it is possible the user is in a windows terminal. 

https://www.wikihow.com/Copy-Files-in-Command-Prompt

File: README.md [hunk 2]
```diff
@@ -103,6 +103,24 @@ For sample implementations, see the [.NET Core Example](https://github.com/sendg
...
+```
+
+### Environment Variable
+
+Update the development environment with your [SENDGRID_API_KEY](https://app.sendgrid.com/settings/api_keys), for example:
+
+```bash
+echo "export SENDGRID_API_KEY='YOUR_API_KEY'" > sendgrid.env
```
Comment (mbernier) [inline README.md]: Can you please update for importing environment variables in windows? 
https://stackoverflow.com/questions/185208/how-do-i-get-and-set-environment-variables-in-c

**Comment by mbernier** [comment]
Two requests and if you can please sign the CLA, as well. Thank you @thepriefy - you've totally kicked ass this #hacktoberfest!

File: .env_sample [hunk 3]
```diff
@@ -0,0 +1 @@
+export SENDGRID_API_KEY=''
```
Comment (mbernier) [inline .env_sample]: please remove export, per https://github.com/sendgrid/smtpapi-php/pull/58

**Comment by thepriefy** [comment]
@mbernier I've done editing but I'm not sure whether it is what you want or not. If you have any change you want or other suggestion, you can tell me and I'll fix it asap. :)

**Comment by mbernier** [comment]
Well done!

**Comment by thinkingserious** [comment]
Hello @thepriefy,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/announcing-hacktoberfest-2017">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**